examples/jupyter/tutorials/tutorial1-ceed.ipynb

```diff
@@ -0,0 +1,337 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Standalone libCEED examples\n",
+    "\n",
+    "This is a tutorial for [libCEED](https://github.com/CEED/libCEED/), the low-level API library for efficient high-order discretization methods developed by the co-design [Center for Efficient Exascale Discretizations](https://ceed.exascaleproject.org/) (CEED) of the [Exascale Computing Project](https://www.exascaleproject.org/) (ECP).\n",
+    "\n",
+    "While libCEED's focus is on high-order finite/spectral element method implementations, the approach is mostly algebraic and thus applicable to other discretizations in factored form, as explained in the [user manual](https://libceed.readthedocs.io/) and [Doxygen documentation](https://codedocs.xyz/CEED/libCEED/)."
+   ]
```

Comment:
It does give a mathematical description and the outputs printed out for the users to see at a first glance :)

Reply:
I guess I'm worried about duplicating the user manual. The more places we have the same data, the more places we have to remember to update changes

Reply:
I know... but @jed yesterday suggested that jupyter-notebooks are great tools to show math. So I thought this was the right place to show that, instead of the python interface tutorials. Let me know if I should get rid of that. FWIW, I don't think we'll ever change the mathematical formulation of ceed-ex1 and ceed-ex2, will we?

Reply:
I suppose I'm confused because I don't see what benefit this has over the .rst file and README.

Reply:
The benefit is to have everything in one file. Again, think of these tutorials as a great tool for outreach about how libCEED works, like live demos and workshops. If you have everything in one file, i.e., the math, the code execution and its output, is easier to demonstrate than having to switch between different windows or for learners having to navigate much to find relevant content.

Reply:
The goal of the notebook is that a user can open it and begin exploring how the objects work. If we just wanted static content evaluated with Sphinx, we wouldn't need Jupyter. But for non-developers to open and begin exploring (locally or by clicking into Binder), Jupyter is really useful.

Reply:
Since this file has no interactive blocks, couldn't we do the same with a static file?

I'm just confused as to how this non-interactive notebook helps, but I don't have strong feelings about excluding it. My other comments are higher priority to me.

Reply:
Do you mean this particular introductory cell is not interactive? Because the execution blocks below of the `ceed-ex1` and `ceed-ex2` examples are indeed interactive. The users can play around with the `-d`, `-m` , `-q `or `-o` parameters and see how this changes the order of accuracy and convergence of the solution.

Reply:
It still feels weird but I guess I see whats going on now.